sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_read.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_read.py
+
+DESCRIPTION:
+    This sample demonstrates how to extract document information using "prebuilt-read"
+    to analyze a given file.
+
+USAGE:
+    python sample_analyze_read.py
+
+    Set the environment variables with your own values before running the sample:
+    1) DOCUMENTINTELLIGENCE_ENDPOINT - the endpoint to your Document Intelligence resource.
+    2) DOCUMENTINTELLIGENCE_API_KEY - your Document Intelligence API key.
+"""
+
+import os
+from utils import get_words
+
+
+def analyze_read():
+    path_to_sample_documents = os.path.abspath(
+        os.path.join(
+            os.path.abspath(__file__),
+            "..",
+            "./sample_forms/forms/Form_1.jpg",
+        )
+    )
+
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.documentintelligence import DocumentIntelligenceClient
+    from azure.ai.documentintelligence.models import DocumentAnalysisFeature
+
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+    document_analysis_client = DocumentIntelligenceClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+    with open(path_to_sample_documents, "rb") as f:
+        poller = document_analysis_client.begin_analyze_document(
+            "prebuilt-read",
+            analyze_request=f,
+            features=[DocumentAnalysisFeature.STYLE_FONT],
+            content_type="application/octet-stream",
```

Comment:
I suppose we'll work to fix this for the next release? So customers dont have to specify the content type when streaming the file

Reply:
Right, it's a bug need to fix in python generator.